robot/impl/resource_manager.go

```diff
@@ -817,6 +817,14 @@ func (manager *resourceManager) markResourceForUpdate(name resource.Name, conf r
 	return nil
 }
 
+func (manager *resourceManager) validateUniqueResource(resources map[string]bool, resourceName string) error {
```

Comment:
```suggestion
func validateUniqueResource(resources map[string]bool, resourceName string) error {
```
[opt/lowprio] (unless there is a reason I am missing?)

Reply:
Sounds good!